host_files/herbert/nginx/sonarqube.zeus.gent

```diff
@@ -0,0 +1,37 @@
+upstream cat {
+   server 10.1.0.210:9000;
+   keepalive 32;
+}
+
+server {
+    listen 80;
+    server_name cat.zeus.gent;
+
+    if ($request_method = GET) {
+        return 301 https://$server_name$request_uri;
+    }
+
+    return 308 https://$server_name$request_uri;
+}
+
+server {
+    listen 443 ssl http2;
+    server_name cat.zeus.gent;
```

Comment:
are you sure this shouldn't be sonarqube.zeus.gent

Reply:
Good point ;)